public/static/partials/user_host_options.html

```diff
@@ -111,7 +112,10 @@
           </ul>
           <div ng-show="!$parent.homeVolumeID">
             <label> Volume size (GB) </label>
-            <input type="number" ng-model="$parent.homeVolumeSize" min="1" max="10000" />
+            <input type="number" ng-disabled="availableVolumeSize() <= 0" ng-model="$parent.homeVolumeSize" min="1" max="availableVolumeSize()" />
```

Comment:
maybe if `availableVolumeSize() <= 0` we shouldn't allow them to spawn a virtual workstation?

Reply:
They can choose an existing volume though. I think the logic of removing "New Volume" is unnecessary and potentially confusing for users; I'd prefer to keep this here to give them more information.

Reply:
but should they be prevented from submitting if they haven't chosen a volume and they've chosen a volume size larger than their allotment?

Reply:
Oh I understand your question. That's fine I can add it.